### PR TITLE
Revive CopyQ plugin

### LINKED ISSUE #137: Any plans to migrate the CopyQ plugin?
My package manager has just updated albert and I seem to be missing a few plugins. The one that I'm going to miss the most is the CopyQ plugin. 

I noticed that you archived this in 55d5f89a7bf67d5ac2cbe28f991aa5e69813f0c7

Do you have plans to add it back to the supported plugins? I presume that in it's archived state it does not support the current plugin API?

### PR DESCRIPTION
**Pull request by BarrensZeppelin**
Fixes #137

I've attempted to do the necessary changes to make the plugin work with the new plugin interface, and it now _works on my machine™_.

It seems like albert does not display item texts as HTML any more, so the effect of underlining the search term in the results is gone.

### CODE REVIEW

**Comment by ManuelSchneid3r** [comment]
Yes I dropped the richtext. Do you mind to volunteer as a maintainer for this plugin?

File: copyq/__init__.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+"""Access CopyQ clipboard.
+
```
Comment (ManuelSchneid3r) [inline copyq/__init__.py:4]: Docsting is intended for exhaustive descritpion. DRY. There are functions for brief desc and synopsis. Rather leave it empty

File: copyq/__init__.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+Synopsis: <trigger> [filter]"""
+
+#  Copyright (c) 2022 Manuel Schneider
+
+import html
+import json
+import re
```
Comment (ManuelSchneid3r) [inline copyq/__init__.py:11]: Please drop all re. They are used for richtet only iirc.

File: copyq/__init__.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    def id(self):
+        return __name__
+
+    def name(self):
+        return md_name
+
+    def description(self):
+        return md_description
+
+    def synopsis(self):
```
Comment (ManuelSchneid3r) [inline copyq/__init__.py:63]: I am not exactly sure but iirc default is exactly filter. May be dropped

File: copyq/__init__.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        return "<filter>"
+
+    def defaultTrigger(self):
+        return "cq "
+
+    def handleQuery(self, query):
+        items = []
+        q_string = query.string
+        pattern = re.compile(q_string, re.IGNORECASE)
```
Comment (ManuelSchneid3r) [inline copyq/__init__.py:72]: Re is not needed

**Review by ManuelSchneid3r — CHANGES_REQUESTED** [review]
Looks good otherwise

File: copyq/__init__.py (hunk 3 above)
Comment (BarrensZeppelin) [reply copyq/__init__.py:59]: Without the override of `synopsis` nothing is displayed with an empty query text.

Comment (ManuelSchneid3r) [reply copyq/__init__.py:59]: Indeed I left it empty. nmv 😅 too much to remember everything

**Review by ManuelSchneid3r — APPROVED** [review]

File: copyq/__init__.py [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+#  Copyright (c) 2022 Manuel Schneider
+
+import json
+import subprocess
+
+from albert import *
+
+md_iid = "0.5"
+md_version = "1.1"
```
Comment (ManuelSchneid3r) [inline copyq/__init__.py:11]: make it 1.2

File: copyq/__init__.py [hunk 6]
```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+#  Copyright (c) 2022 Manuel Schneider
```
Comment (ManuelSchneid3r) [inline copyq/__init__.py:3]: remove this one

File: copyq/__init__.py [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+import json
+import subprocess
+
+from albert import *
+
+md_iid = "0.5"
+md_version = "1.1"
+md_name = "CopyQ"
+md_description = "Access CopyQ clipboard"
+md_url = "https://github.com/albertlauncher/python"
```
Comment (ManuelSchneid3r) [inline copyq/__init__.py:14]: theres no md_license maybe something like bsd-2-clause or mit?

**Pull request merged**